android/app/src/main/java/com/mattermost/helpers/push_notification/Post.kt

```diff
@@ -73,9 +73,12 @@ internal suspend fun PushNotificationDataRunnable.Companion.fetchPosts(
                         if (userId != null && userId != currentUserId && !userIdsAlreadyLoaded.contains(userId) && !userIds.contains(userId)) {
                             userIds.add(userId)
                         }
-                        val message = post?.getString("message")
-                        if (message != null) {
-                            val matchResults = regex.findAll(message)
+                        fun findNeededUsernames(text: String?) {
```

Comment:
Why are we re-declaring the function on every iteration?

Reply:
Because I thought I needed some variable in the closure that were created on the loop :facepalm: Will relocate it.